changelogs/current.yaml

```diff
@@ -35,6 +35,10 @@ minor_behavior_changes:
     AWS request signing and AWS Lambda extensions will now no longer return empty credentials (and fail to sign) when
     credentials are still pending from the async credential providers. If all providers are unable to retrieve credentials
     then the original behaviour with a signing failure will occur.
+- area: formatter
+  change: |
+    The formatter ``%CEL%`` and ``%METADATA%`` will be treated as built-in formatters and could be used directly in the
+    substitution format string if the related extensions are linked.
```

Comment:
What does it mean that they will be enabled by default? 
From the release note I assume they are not linked by default. If they are not linked, would the config will still be valid (but not emit any log)?

Reply:
They are linked by default. But if you want to enable them and use them at substitution format string, you need to configure them at the `SubstitutionFormatString.formatters`. (And it's impossible to use them for the header mutation in the previous implementation because the header mutation API doesn't contains `formatters` extension point).

Now, you needn't to configure them at the `SubstitutionFormatString.formatters` and the `CEL/METADATA` could be used in the substitution format string directly at every where.

Reply:
I think the users of `CEL`/`METADATA` should be clear to that context.

Reply:
> It's unclear to me then why are these extensions (if they are supported by default). If I understand correctly, the intent of the PR is make the use of the CEL and METADATA require one less configuration knob. IMHO if they must be supported then one should consider moving them to core, but don't feel strongly either way.

Actually, I also don't know why. IMO, all these should be `untyped extensions`. Extensions make they are extendable. The forks could register their own formatters. But proto configurations should be unnecessary because the formatter self could use addtional command parameters as configuration.